app/views/idv/textquestion.html.slim

```diff
@@ -0,0 +1,17 @@
+- title t('upaya.titles.idv.p2')
+
+.clearfix.mxn2
+  .sm-col-7.px2.mx-auto
+    .panel
+      .panel-heading
+        h2.m0 = 'Verify identity'
+      .mb1.h6.caps Question 1
+      =form_tag({ controller: 'idv', action: 'answer' }, method: 'post')
+        =hidden_field_tag('question_key', quest.key)
+        =hidden_field_tag('text_question', true)
+        h3.mt2.mb2
+          =quest.display
+        .sm-col.sm-col-12.px1
```

Comment:
this line should be removed (and the next line should be unindented) -- it's unneeded and adding a padding issue